mypy/checker.py

```diff
@@ -81,13 +106,13 @@ def __init__(self) -> None:
         self.frames = []  # type: List[Frame]
         # The first frame is special: it's the declared types of variables.
         self.frames.append(Frame())
+        # We want a second frame that we can update.
+        self.frames.append(Frame())
```

Comment:
I notice this second frame's parent is not the first frame. Is the first frame so special that it should just be a separate field of the binder object?